client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-footer.jsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { actionType } from './constants';
+
+class DomainConnectAuthorizeFooter extends Component {
+
+	placeholder = () => {
+		return (
+			<div className="domain-connect__footer is-placeholder">
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	renderActionConfirmCancel = () => {
+		return (
+			<div>
+				<p>
+					{
+						translate( 'When you\'re ready to proceed, click Confirm. If this isn\'t what you meant to do, ' +
+							'click Cancel and we won\'t make any changes.' )
+					}
+				</p>
+				<div>
+					<Button
+						busy={ actionType.READY_TO_SUBMIT !== this.props.showAction }
+						className="domain-connect__button"
+						disabled={ actionType.READY_TO_SUBMIT !== this.props.showAction }
+						icon
+						onClick={ this.props.onConfirm }
+						primary>
+						<Gridicon icon="checkmark" /> { translate( 'Confirm' ) }
+					</Button>
+					<Button
+						busy={ actionType.READY_TO_SUBMIT !== this.props.showAction }
+						className="domain-connect__button"
+						disabled={ actionType.READY_TO_SUBMIT !== this.props.showAction }
+						icon
+						onClick={ this.props.onClose }>
+						<Gridicon icon="cross" /> { translate( 'Cancel' ) }
+					</Button>
+				</div>
+			</div>
+		);
+	}
+
+	renderActionClose = () => {
+		return (
+			<div>
```

Comment:
Um...the actual answer is probably ignorance; but, again, I'm going to blame it on multiple iterations of this code that I didn't clean up well enough. 😄